Source/WebKit/UIProcess/mac/DisplayLink.h

```diff
@@ -27,70 +27,71 @@
 
 #if HAVE(CVDISPLAYLINK)
 
-#include "Connection.h"
 #include "DisplayLinkObserverID.h"
 #include <CoreVideo/CVDisplayLink.h>
 #include <WebCore/AnimationFrameRate.h>
 #include <WebCore/DisplayUpdate.h>
 #include <WebCore/PlatformScreen.h>
-#include <wtf/HashMap.h>
+#include <wtf/HashSet.h>
```

Comment:
Why this change is necessary? This class is only using HashMap.

Reply:
Leftover from a previous version of the patch.